Misc fixes / updates

tas50 opened the pull request:
1 per commit

cwjohnston commented on .gitignore:
```diff
@@ -1,17 +1,50 @@
+.idea
+
+#chef stuff
+Berksfile.lock
```
At the moment we're still using librarian-chef in this project. Will you please add `Berkshelf` to gitignore as well? Looks like that line was lost in this revision.

cwjohnston commented:
Looks like the version of test-kitchen in our bundle does not know about the centos platforms added here:

```
>>>>>> ------Exception-------
>>>>>> Class: Kitchen::ActionFailed
>>>>>> Message: Failed to complete #create action: [Expected process to exit with [0], but received '1'
---- Begin output of vagrant up --no-provision --provider virtualbox ----
STDOUT: Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'centos-7.2' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Box file was not detected as metadata. Adding it directly...
==> default: Adding box 'centos-7.2' (v0) for provider: virtualbox
    default: Downloading: centos-7.2

STDERR: An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

Couldn't open file /Users/cwj/github/sensu-chef/.kitchen/kitchen-vagrant/kitchen-sensu-chef-sysv-centos-72/centos-7.2
---- End output of vagrant up --no-provision --provider virtualbox ----
Ran vagrant up --no-provision --provider virtualbox returned 1]
>>>>>> ----------------------
```

Will you please adjust the platform configurations so they will run under the bundled version of test-kitchen? Might just need a `box_name` specified.

tas50 commented:
I added Berkshelf back to the ignore file and updated the version of Test Kitchen.  There's been a ton of Test Kitchen work lately, particular Windows work.

cwjohnston commented:
Thanks Tim. 

FWIW, I have tried to use the WinRM transport in newer versions of test-kitchen, but it seems like its not possible to install `netfx35` feature (.NET Framework 3.5) via WinRM. See https://github.com/sensu/sensu-chef/issues/423 for some background. Maybe you or someone at Chef has some specific experience with this and could give us some pointers?

tas50 commented:
@cwjohnston Let me check with the Windows folks.  WinRM itself has a ton of issues that have caused us a lot of pain in Test Kitchen land.

cwjohnston commented:
@tas50 were you able to learn anything from your team re: installing .NET during a test-kitchen run? I would love to be free of our existing hacks.

tas50 commented:
It's broken due to WinRM limitations, but you can work around that by using the chef zero scheduled task.  There's some work in core Test Kitchen to bring that code in to better support the various things that refuse to run via WinRM.

https://github.com/smurawski/chef-zero-scheduled-task

cwjohnston commented:
I integrated many of the changes in this PR via #470. Closing.

tas50 commented:
Thanks for putting together a working version of the PR
